frontend/src/components/Forms/OtherForm/index.js

```diff
@@ -0,0 +1,22 @@
+import { PropTypes } from "prop-types";
+import Input from "components/Input";
+
+const OtherForm = ({ extraInterest, handleChange }) => {
+  return (
+    <Input
+      id="other"
+      name="Other interest"
+      type="text"
+      placeholder="What are you interested in?"
+      value={extraInterest}
+      onChange={handleChange("extraInterest")}
+    />
+  );
+};
+
+OtherForm.propTypes = {
+  extraInterest: PropTypes.string.isRequired,
+  handleChange: PropTypes.func.isRequired,
+};
+
+export default OtherForm;
```

Comment:
Since you're using only the `Input` component now, you might not need this extra component anymore. You could use `Input` directly in the `SignupPage`.